db/schema.rb

```diff
@@ -78,6 +78,8 @@
     t.string "github_og_image"
     t.string "github_og_url"
     t.string "twitter_handle"
+    t.string "mastodon_url"
+    t.string "bluesky_handle"
```

Comment:
How did you make these database changes, if not with a database migration?

Knowing that you're new to rails, you may way to read through this guide: https://guides.rubyonrails.org/active_record_migrations.html

-----

In theory you should have been able to generate a migration for this with something like this one-liner (run from the terminal)

```bash
rails generate migration AddMastodonUrlBlueskyHandleToParticipants mastodon_url:string bluesky_handle:string
```